templates/pylibrary/{{cookiecutter.module_name}}/README.md

```diff
@@ -34,3 +34,9 @@ see above) as follows:
 ```
 python3 -m pytest tests/
 ```
+
+Execute the library's CI locally with [act](https://github.com/nektos/act) as follows:
+
+```
+act -j ci-libs-{{ cookiecutter.package_name }}
```

Comment:
I see `ci-libs-{MODULE_NAME}:`

https://github.com/tweag/python-monorepo-example/pull/11/files#diff-0a906118d18963050e5986513bc370e6922f5dcce818e361c5d48f004637a8a8R22

but here it says `package_name`

is it okay?